models/articles.model.js

```diff
@@ -72,3 +72,22 @@ exports.updateArticleById = (inc_votes, article_id) => {
       }
     });
 };
+
+exports.fetchCommentsByArticleId = (article_id) => {
+  return db
+    .query(
+      `SELECT comment_id, votes, created_at, author, body 
+    FROM comments
+    WHERE article_id = $1 ORDER BY created_at DESC;`,
+      [article_id]
+    )
+    .then((articles) => {
+      if (!articles.rows.length) {
```

Comment:
So it is at this point that we would need to check for whether than article actually exists. Because if the article did exist but had no comments, it would still return an empty array. I would argue though, that a 404 in that specific would be the wrong status code as it's a valid request. So it should return a 200